### PR TITLE
[pillow] Pillow fuzzers

### PR DESCRIPTION
**Pull request by guidovranken**
Placeholder, do not merge yet.. It's been finding a whole bunch of bugs locally that I am currently reporting. Publishing the fuzzer and corpus right now would divulge these bugs ahead of the release, so we will need to wait for fixes first.

### CODE REVIEW

**Comment by Dor1s** [comment]
We accept the project, please ping this PR when it's ready for review.

File: projects/pillow/project.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+homepage: "https://python-pillow.org/"
+primary_contact: "guidovranken@gmail.com"
```
Comment (radarhere) [inline projects/pillow/project.yaml]: Would you also like to cc someone at Pillow?

Comment (guidovranken) [reply projects/pillow/project.yaml]: I finally got around to this. Please write 1 or more Pillow CC's.

Comment (radarhere) [reply projects/pillow/project.yaml]: security@python-pillow.org thanks

Comment (guidovranken) [reply projects/pillow/project.yaml]: Is this address linked to a Google account, or can you do that? If not, you will get e-mail notifications, but cannot see bug report contents.

**Comment by guidovranken** [comment]
It's ready to be merged as far as I'm concerned. @Dor1s

File: projects/pillow/project.yaml (hunk 1 above)
Comment (radarhere) [reply projects/pillow/project.yaml]: If you'd like this to be merged, then we can at least get the e-mail notifications with that address, and can return to it later if we so choose. Thanks

Comment (guidovranken) [reply projects/pillow/project.yaml]: Sure.

**Pull request merged**